chart/compass/README.md

```diff
@@ -12,12 +12,12 @@ The Compass consists of the following sub-charts:
 
 ## Details
 
-To learn more about the Compass, see the [Overview](https://github.com/kyma-incubator/compass#Overview) document.
+To learn more about the Compass, see the [Overview](https://github.com/kyma-incubator/compass#overview) document.
 
 ## Configuration
 
 | Parameter | Description | Values | Default |
 | --- | --- | --- | --- |
 | `database.useEmbedded` | Specifies whether `postgresql` chart should be installed | true/false | `true` |
 
-To learn how to use managed GCP database see [that document](../../docs/configure-managed-gcp-postgresql.md)
\ No newline at end of file
+To learn how to use managed GCP database see [this document](https://github.com/kyma-incubator/compass/blob/master/docs/configure-managed-gcp-postgresql.md)
```

Comment:
Why do you always refer to the document from the master branch? Shouldn't this path be a relative one?

Reply:
Relative path leading outside chart directory won't work for example when chart is put in kyma.